### ISSUE TITLE
predict = tf.argmax(tf.nn.softmax(logits), axis=1, name="predictions")  这里axis

### ISSUE THREAD

**Issue by 452896915** [huwenxianglyy/bert-use-demo#7]

**Comment by 452896915** [comment]
predict = tf.argmax(tf.nn.softmax(logits), axis=1, name="predictions")  这里是不是应该axis=-1 或 axis = 2

**Comment by 452896915** [comment]
因为logits的size是(batch_size, max_seq_len, label_size)
